main.go

```diff
@@ -65,21 +65,17 @@ func cmdWithdraw(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("key %s not found in config", keyName)
 	}
 
+	denom, err := getDenom(conf, chainName)
+	if err != nil {
+		return fmt.Errorf("denom not found in config or chain registry: %s", err)
+	}
+
 	nodeAddress := chain.Node
 	if flagNode != "" {
 		nodeAddress = flagNode
 	}
 
-	isDenomSet := cmd.Flags().Changed("denom")
```

Comment:
Shouldn't we be able to override this from the command-line too?

We don't have many command-line parameters and I'm not sure what the best way to pursue: only use the config file for parameters or allow the command-line to override things too?

Reply:
good point, I think it makes sense to have an option to override. One use case for that I believe it will be possible to pay fees with another denom other than native. So overriding would make sense in this case too.